examples/conways_game_of_life/conways_game_of_life/server.py

```diff
@@ -1,12 +1,32 @@
-from mesa.visualization.modules import CanvasGrid
-from mesa.visualization.ModularVisualization import ModularServer
+from mesa.visualization.VegaVisualization import VegaServer
+from mesa.visualization.UserParam import UserSettableParameter
 
-from .portrayal import portrayCell
-from .model import ConwaysGameOfLife
 
+from .model import ConwaysGameOfLife
 
-# Make a world that is 50x50, on a 250x250 display.
-canvas_element = CanvasGrid(portrayCell, 50, 50, 250, 250)
+grid_spec = """
+{
+    "$schema": "https://vega.github.io/schema/vega-lite/v3.json",
```

Comment:
Is `$schema` a standard naming convention? (I am not a fan of the `$`.)

Reply:
Yes, this is part of the JSON schema specifications. It's a special keyword and for example picked up by editors/IDEs to provide autocompletion